### ISSUE TITLE
Add a command to prune all generated/built files from a checkout

### ISSUE THREAD

**Issue by ivanlei** [Agoric/Agoric#7180]
## What is the Problem Being Solved?
If a build fails (or even if it doesn't), the checkout will have numerous output files, `node_modules /` directories, built binary packages, etc. A command to prune all generated/built files from a checkout will help to minimize the chance that build artifacts on disk prevent further builds from succeeding.

## Description of the Design
The command should either add to `yarn clean` or be some sort of `yarn super-duper-clean`

## Security Considerations
Should be none

## Scaling Considerations
Should be none

**Comment by arirubinstein** [comment]
perhaps `git clean -dfx` ?

**Comment by mhofman** [comment]
Indeed, I often run `git clean -xfd && git submodule foreach --recursive git clean -xfd`
